eng/tools/spec-gen-sdk-runner/src/types.ts

```diff
@@ -29,3 +29,14 @@ export type VsoLogs = Map<
     warnings?: string[];
   }
 >;
+
+/**
+ * Represents the result of the spec-gen-sdk generation process.
+ */
+export interface SpecGenSdkArtifactInfo {
+  managementPlane: boolean;
```

Comment:
Should this just be type so there is no chance it can have a conflict where both mgmt and data plane are set to try?

Reply:
It's intended to be separated field as a spec PR can include both management plane specs and data plane specs.